File: 242-valid-anagram/valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) != len(t):
            return False
        for item in set(s):
            if t.count(item) != s.count(item):
                return False

        return True
        

File: 242-valid-anagram/valid-anagram.rs
impl Solution {
    pub fn is_anagram(s: String, t: String) -> bool {
        let mut sset = Vec::new();
        let mut tset = Vec::new();
        for c in s.chars(){
            sset.push(c);
        }
        for c in t.chars(){
            tset.push(c);
        }
        sset.sort();
        tset.sort();
        let ret = sset == tset;
        ret
    }
}